cypress/e2e/audio-player/audio-player.spec.ts

```diff
@@ -0,0 +1,563 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+import { SettingLevel } from "../../../src/settings/SettingLevel";
+import { Layout } from "../../../src/settings/enums/Layout";
+
+describe("Audio player", () => {
+    let homeserver: HomeserverInstance;
+    let roomId: string;
+    const TEST_USER = "Hanako";
+
+    const percyCSS =
+        // FIXME: hide mx_SeekBar because flaky - see https://github.com/vector-im/element-web/issues/24898
+        ".mx_SeekBar, " +
+        ".mx_JumpToBottomButton, .mx_MessageTimestamp, .mx_RoomView_myReadMarker { visibility: hidden !important; }";
+
+    const visitRoom = () => {
+        cy.visit("/#/room/" + roomId);
+        cy.setSettingValue("layout", null, SettingLevel.DEVICE, Layout.Group);
+
+        // Wait until configuration is finished
+        cy.contains(
+            ".mx_RoomView_body .mx_GenericEventListSummary[data-layout='group'] .mx_GenericEventListSummary_summary",
+            "created and configured the room.",
+        ).should("exist");
+    };
+
+    const uploadFile = (file: string) => {
+        // Upload a file from the message composer
+        cy.get(".mx_MessageComposer_actions input[type='file']").selectFile(file, { force: true });
+
+        cy.get(".mx_Dialog").within(() => {
+            // Click "Upload" button
+            cy.get("[data-testid='dialog-primary-button']").should("have.text", "Upload").click();
+        });
+
+        // Wait until the file is sent
+        cy.get(".mx_RoomView_statusArea_expanded").should("not.exist");
+        cy.get(".mx_EventTile.mx_EventTile_last .mx_EventTile_receiptSent").should("exist");
+    };
+
+    const checkPlayerFilenameLong = () => {
+        // Detect the audio file
+        cy.get(".mx_EventTile_mediaLine .mx_MAudioBody").within(() => {
+            // Assert that the audio player is rendered
+            cy.get(".mx_AudioPlayer_container").within(() => {
+                // Assert that media information is rendered
+                cy.get(".mx_AudioPlayer_mediaInfo").within(() => {
+                    cy.get(".mx_AudioPlayer_mediaName").should("have.text", "1sec-long-name-audio-file.ogg");
+                    cy.contains(".mx_AudioPlayer_byline", "00:01").should("exist");
+                    cy.contains(".mx_AudioPlayer_byline", "(3.56 KB)").should("exist"); // actual size
+                });
+
+                // Assert that the play button is rendered
+                cy.get("[data-testid='play-pause-button'][aria-label='Play']").should("exist");
+            });
+        });
+    };
+
+    // Take snapshots in modern and bubble layout, outputting log for reference/debugging
+    // Player on IRC layout should have the same layout as group layout
```

Comment:
I think some clarification would be helpful here:

```suggestion
    // We don't test IRC layout, since it should be the same as group layout.
```

... if my understanding is correct?

Reply:
Right, that is correct. Thanks!